File: data/src/main/java/js/task/data/local/db/model/DailyMotionPage.java
package js.task.data.local.db.model;

import java.util.List;

public class DailyMotionPage
{
    Integer page;
    Integer limit;
    Boolean explicit;
    Integer total;
    Boolean has_more;
    List<DailyMotion> list;

    public Integer getPage()
    {
        return page;
    }

    public Integer getLimit()
    {
        return limit;
    }

    public Boolean getExplicit()
    {
        return explicit;
    }

    public Integer getTotal()
    {
        return total;
    }

    public Boolean getHasMore()
    {
        return has_more;
    }

    public List<DailyMotion> getList()
    {
        return list;
    }
}

File: data/src/main/java/js/task/data/local/db/model/DailyMotionPage.kt
package js.task.data.local.db.model

import com.google.gson.annotations.SerializedName

data class DailyMotionPage
(
    val page : Int?,
    val limit : Int?,
    val explicit : Boolean?,
    val total : Int?,
    @SerializedName("has_more")
    val hasMore : Boolean?,
    val list : List<DailyMotion?>?,
)